python-frontend/src/test/java/org/sonar/python/semantic/FullyQualifiedNameTest.java

```diff
@@ -75,6 +75,15 @@ public void submodule_alias() {
     assertNameAndQualifiedName(tree, "fn", "mod.submod.fn");
   }
 
+  @Test
+  public void alias_import_preserves_fqn() {
```

Comment:
I think we can add a comment with a link to the typeshed file, to give more context to this unit test